root/io/io/MainEventCompressionBenchmark.cxx

```diff
@@ -0,0 +1,119 @@
+#include "TFile.h"
+#include "TTree.h"
+
+#include "benchmark/benchmark.h"
+#include "rootbench/RBConfig.h"
+
+static std::string GetAlgoName(int algo) {
+   std::string algo_name = "";
+   if (algo == 0) {
```

Comment:
0 means actually default compression algorithm (which is zlib in this case). For zlib it is 1. Can you add please  "algo ==1" as a conditional statements? I think here you can use switch statement easily.